images/build/php/build.sh

```diff
@@ -0,0 +1,162 @@
+#!/bin/bash
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT license.
+# --------------------------------------------------------------------------------------------
+
+set -eux
+
+START_TIME=`date +%s`
```

Comment:
Did we take this script from somewhere? I see there's lots of references to GitHub issues. If yes, then we should reference the source

Reply:
Will add a SOURCE.md file